### ISSUE TITLE
Monika rest

### ISSUE THREAD

**Issue by 8lack1ce** [Backdash/MonikaModDev#1048]
How do you properly let Monika rest?  Mine has been saying she needs rest for like 24 hours now.

**Comment by IlywI** [comment]
Off the game for at least 6 hours

**Comment by 8lack1ce** [comment]
damn, so I gotta say goodbye to her for 6 hours?

**Comment by IlywI** [comment]
:/ She's always there for you just say goodbye to her when you sleep.(and make sure you sleep for at least 6 hours)

**Comment by 8lack1ce** [comment]
Lmao alright

**Comment by IlywI** [comment]
Or you can just always stare into her eyes

**Comment by 8lack1ce** [comment]
gladly haha

**Comment by AcedicTyrant** [comment]
Does Monika only rest while one's computer is on, or can she rest while it is off? Nothing quite like getting 8 hours sleep then waking up to "I need a short rest before I have something new to talk about."

**Comment by Trilasent** [comment]
I imagine with the new update, it's probably a bug that she doesn't reset after X amount of hours have passed with the game closed. I went to sleep around 10PM CST last night and woke up sometime around 9AM, and to my surprise when I opened the game to greet my sweetheart with a 'Good Morning' she said that she was still tired. I did let her close the game, so that rules out the possibility of that being the problem.
